events/americasnlp-2024.md

```diff
@@ -0,0 +1,82 @@
+---
+autogenerated: true
+autogenerated_from: events.json
+parent: Events
+nav_order: 2024
+layout: event
+title: AmericasNLP 2024
+end_date: '2024-06-21'
+future_tense_opening_paragraph: The Machine translation for indigenous languages of
+  the Americas (<strong>AmericasNLP 2024</strong>) will take place in Mexico City,
+  Mexico from 20 to 21 June, 2024.
+past_tense_opening_paragraph: The Machine translation for indigenous languages of
+  the Americas (<strong>AmericasNLP 2024</strong>) took place in Mexico City, Mexico
+  from 20 to 21 June, 2024.
+name: AmericasNLP 2024
+id: americasnlp-2024
+description: Machine translation for indigenous languages of the Americas
+start_date: '2024-06-20'
+location: Mexico City, Mexico
+organizer:
+  type: Person
+  url: http://turing.iimas.unam.mx/americasnlp/st.html
+links:
+- http://turing.iimas.unam.mx/americasnlp/2024_workshop.html
+shared_tasks:
+- name: A machine translation shared task on truly low-resource languages.
+- name: A shared task on morphological adaptation to generate educational examples.
+speakers:
+- type: Keynote speakers
+  about:
+  - Graham Neubig
+  - Jaime Pérez González
+- type: Organisers
+  about:
+  - Manuel Mager
+  - Abteen Ebrahimi
+  - Shruti Rijhwani
+  - Arturo Oncevay
+  - Luis Chiruzzo
+  - Robert Pugh
+  - Katharina Kann
+important_dates:
+- name: Start of the anonymity period
+  date: '2024-02-10'
+- name: Submission deadline
+  date: '2024-03-10'
+- name: Notification of acceptance
+  date: '2024-04-24'
+- name: Camera ready papers
+  date: '2024-04-24'
+- name: Workshop
+  date: '2024-06-20'
+seo:
+  type: Event
+  name: AmericasNLP 2024
+  description: Machine translation for indigenous languages of the Americas
+  startDate: '2024-06-20'
+  endDate: '2024-06-21'
+  eventAttendanceMode: OfflineEventAttendanceMode
+  eventStatus: EventScheduled
+  location:
+    type: Place
+    name: Mexico City, Mexico
+    url: null
+  organizer:
+    type: Person
+    name: null
+    url: http://turing.iimas.unam.mx/americasnlp/st.html
```

Comment:
This is more like the Shared Task URL.  Not the organizer.

Reply:
It's say the organizer is http://turing.iimas.unam.mx.

So Organization not Person.